react/src/App.jsx

```diff
@@ -27,12 +28,14 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="/about-us" element={<AboutUs />} />
         <Route path="/calendar" element={<Calendar />} />
-        <Route path="/login" element={<Login onLogin={handleLogin} />} />
+        {/* <Route path="/login" element={<Login onLogin={handleLogin} />} /> */}
+        <Route path="/login" element={<Construction/>} />
         <Route path="/navigate" element={<Navigate />} />
-        <Route path="/register" element={<Register />} />
+        <Route path="/register" element={<Construction/>} />
         <Route path="/register-org" element={<RegisterOrg />} />
-        <Route path="/contact-us" element={<ContactUs />} />
+        <Route path="/contact-us" element={<Construction/>} />
         <Route path="*" element={<NotFound />} />
+        <Route path="/under-construction" element={<Construction/>} />
```

Comment:
Or you could not have added this and just swapped out the elements for register and contact-us for Construction like you already did. In this case I would have left the NavLinks alone as they would have routed to the same page anyways. This also would simplify turning them back on when they are complete.